packages/loopring_v3/contracts/aux/access/LoopringIOExchangeOwner.sol

```diff
@@ -201,6 +206,8 @@ contract LoopringIOExchangeOwner is SelectorBasedAccessManager, ChiDiscount, ERC
     {
         uint cursor = 0;
 
+        // Reuse the data when possible to save on some memory alloc gas
+        bytes memory txsData = new bytes(ExchangeData.TX_DATA_AVAILABILITY_SIZE*6);
```

Comment:
why `*6` instead of `*txCallback.numTxs`

Reply:
This it just the initial memory allocation for the txsData, and for the AMM pools we need 6 transactions so I just set that as the default. But could be made a bit more general perhaps by checking the first array element there instead (or just a bytes array with length 0).